wezterm-gui-subcommands/src/lib.rs

```diff
@@ -146,6 +146,7 @@ mod test {
 
 #[derive(Debug, Parser, Default, Clone)]
 #[command(trailing_var_arg = true)]
+#[command(short_flag_alias = 'e')]
```

Comment:
```suggestion
#[command(visible_short_flag_alias = 'e')]
```

seems to make it show up in the `--help` output:

```
; ./target/debug/wezterm --help
Wez's Terminal Emulator
http://github.com/wez/wezterm

Usage: wezterm [OPTIONS] [COMMAND]

Commands:
  start                  Start the GUI, optionally running an alternative program [aliases: -e]
```

Reply:
The way clap appends that alias is slick!